keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -257,27 +255,44 @@ Metadata:
   finalizers:
   - cosi.io/finalizer [3]
 spec:
-  bucketClassName: [4]
-  bucketRef: [5]
+  provisioner: [4]
+  releasePolicy: [5]
+  accessModes: [6]
+  supportedProtocols: [7]
+  bucketClassName: [8]
+  bucketRef: [9]
     name:
     namespace:
-  secretRef: [6]
+    uuid:
+    resourceVersion:
+  secretRef: [10]
     name:
     namespace:
-  objectBucketName: [7]
+  bucketIdentifier: [11]
+  parameters: 
 status:
-  bucketAttributes: <map[string]string> [8]
-  phase: [9]
+  message: [12]
+  phase: [13]
   conditions:
 ```
 1. `name`: Generated in the pattern of `<BUCKET-CLASS-NAME>'-'<RANDOM-SUFFIX>`. 
 1. `labels`: COSI controller adds the label to its managed resources for easy CLI GET ops.  Value is the driver name returned by GetDriverInfo() rpc. Characters that do not adhere to [Kubernetes label conventions](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set) will be converted to ‘-’.
 1. `finalizers`: COSI controller adds the finalizer to defer Bucket deletion until backend deletion ops succeed.
+1. `provisioner`: The name of the drive as defined in the BucketClass.  Used by sidecars to filter BucketContents.
```

Comment:
maybe add `adhering to DNS naming conventions, eg. "ceph-rgw.cosi.redhat.com"`?

Reply:
👍